_pytest/doctest.py

```diff
@@ -145,7 +143,6 @@ def get_optionflags(parent):
 
 
 class DoctestTextfile(DoctestItem, pytest.Module):
-
```

Comment:
Those whitespace changes are unrelated, and wrong according to [pep8](https://www.python.org/dev/peps/pep-0008/#blank-lines)